### PR TITLE
Nav our story

### PR DESCRIPTION
**Pull request by DJCPJ**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+#our_story h2 {
+    font-family: 'lobster', cursive;
+    color: #f8512f;
+    font-size: 6rem;
+    text-shadow: 1px 1px 5px #0a0a0a;
+
+}
+
+
+#OurStory h3 {
```
Comment (ayojackay) [inline css/styles.css:139]: I think this should be `our_story`.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+ }
+
+ .carousel-capation p {
+     color: #f7f7f7;
+     font-size: 16px; 
+ }
+
+ /* OUR STORY */
+
+ #OurStory .row {
```
Comment (ayojackay) [inline css/styles.css:126]: I think this should be `our_story`.

**Review by ayojackay — APPROVED** [review]
It looks great. just had a question on how you wrote out the id: `our_story`. Great job!

File: css/styles.css (hunk 1 above)
Comment (DJCPJ) [reply css/styles.css:139]: Yes just changed it had to make a change in html as well.

**Pull request merged**